packages/ods/src/components/password/src/components/ods-password/ods-password.tsx

```diff
@@ -118,6 +118,8 @@ export class OdsPassword {
   }
 
   private async onOdsChange(event: OdsInputChangeEvent): Promise<void> {
+    event.stopPropagation();
```

Comment:
did you try if this works it you have an onOdsChange handler on the host also (or if you need stopImmediateProp)?
like:
```
<odsPassword onOdsChange={ ... } />
```
Ensure that the handler is only called with the expected password event